support-models/src/test/scala/com/gu/support/workers/SerialisationSpec.scala

```diff
@@ -51,6 +52,25 @@ class SerialisationSpec extends AsyncFlatSpec with SerialisationTestHelpers with
     testDecoding[SendThankYouEmailState](thankYouEmailJson())
   }
 
+  it should "roundtrip successfully" in {
```

Comment:
not sure if this test is in the right place or even needed - the others are all using hard coded json